### PR TITLE
Update skypy notebook

### PR DESCRIPTION
**Pull request by Vikramb1**

### CODE REVIEW

File: Notebooks/Simulations/skypy_lenstronomy.ipynb [hunk 1]
```diff
@@ -29,7 +29,7 @@
      "name": "stdout",
```
Comment (sibirrer) [inline Notebooks/Simulations/skypy_lenstronomy.ipynb:33]: <p>Can you update the versions of SkyPy and lenstronomy for this notebook, and also add your name to the Authors ;)</p><p> </p><p>Can you also add a link to the SkyPy examples?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='0'/>

Comment (sibirrer) [inline Notebooks/Simulations/skypy_lenstronomy.ipynb:33]: <p>I spotted a typo 'th g,r,i'...</p><p> </p><p>Perhaps we can add some more links to the SimulationAPI classes</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (sibirrer) [inline Notebooks/Simulations/skypy_lenstronomy.ipynb:33]: <pre class="ql-syntax" spellcheck="false">Line #29.            ax.imshow(img, aspect='equal', origin='lower')
</pre><p>One thing that I generally wonder is whether we can get more colorful pictures showing more colors from blue, green over yellow to red</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='14'/>

**Review by sibirrer — CHANGES_REQUESTED** [review]
Looks great!
I added some further suggestions to improve the notebook documentation. One main point is also whether you understand what is happening there and if not, you should ping me to add further documentation.

**Comment by Vikramb1** [comment]
> Looks great! I added some further suggestions to improve the notebook documentation. One main point is also whether you understand what is happening there and if not, you should ping me to add further documentation.

I think I understand all of it because of the work I had done using the simAPI notebook. I have updates the pr with the changes you requested. Please let me know if there is anything else you want me to add/modify!

**Review by sibirrer — APPROVED** [review]
Looks all great!
I look into the colors. There might be a bug how things get handled as lens and source seem to have the same colors

**Pull request merged**